resources/js/views/general/Tag.vue

```diff
@@ -168,6 +168,11 @@ export default {
     },
     computed: {
 
+        //whether the user wants to show previous tags. return type: boolean
+        show_previous_tags(){
+            return this.user.show_previous_tags == 1;
+        },
```

Comment:
Function can be updated similar to the other. Do we need to duplicate them? We could pass the value from one component to the other and accept is as a prop

Reply:
I got rid of the duplicate. I forgot to get rid of it in a previous commit.